src/UnityExtension/Assets/Editor/GitHub.Unity/UI/Window.cs

```diff
@@ -158,6 +162,12 @@ public override void OnUI()
 
             DoToolbarGUI();
 
+            if (nextTab.HasValue)
+            {
+                SetActiveTab(nextTab.Value);
```

Comment:
This can't be done here, SetActiveTab calls OnDisable and OnEnable on the views, and they're not set up to expect these events multiple times (OnUI is called multiple times, one for each phase of the render pass). Also these events should be called outside the UI phase in general, preferably before OnDataUpdate is called on the views (which happens before OnUI is called).

TL;DR, the order of events for child views should be "OnDisable" (the old one) -> "OnEnable" (the new one) -> "OnDataUpdate" -> "OnGUI"